core/templates/dev/head/pages/story_editor/story_editor.html

```diff
@@ -33,8 +33,11 @@
 {% block footer_js %}
   {{ super() }}
   <script src="/templates/dev/head/mathjaxConfig.js"></script>
+  <script src="/third_party/static/ckeditor-4.9.2/ckeditor.js"></script>
```

Comment:
Reorganize this so that the order is:
- ckeditor
- mathjaxConfig
- MathJax

Reply:
ok

Reply:
done